src/main/views/MattermostView.js

```diff
@@ -117,12 +118,12 @@ export class MattermostView extends EventEmitter {
     loadRetry = (loadURL, err) => {
         this.retryLoad = setTimeout(this.retry(loadURL), RELOAD_INTERVAL);
         WindowManager.sendToRenderer(LOAD_RETRY, this.server.name, Date.now() + RELOAD_INTERVAL, err.toString(), loadURL.toString());
-        log.info(`[${this.server.name}] failed loading ${loadURL}: ${err}, retrying in ${RELOAD_INTERVAL / SECOND} seconds`);
+        log.info(`[${Util.shorten(this.server.name)}] failed loading ${loadURL}: ${err}, retrying in ${RELOAD_INTERVAL / SECOND} seconds`);
```

Comment:
Is it as big of a deal to shorten the name in the console logs?

Reply:
not a big deal, but it was really annoying to have to chase the lines